README.rst

```diff
@@ -141,6 +141,15 @@ Please note the following guidelines for contributing:
   * If you are adding new functionality, you must include basic tests
     and documentation.
 
+
```

Comment:
all right. I'll discard all the changes I made to the README. Other than that is the pull request ok then?